project/src/main/scala/org/broadinstitute/clio/sbt/Docker.scala

```diff
@@ -35,7 +37,17 @@ object Docker {
     new Dockerfile {
       from("openjdk:8")
       label("CLIO_VERSION", version.value)
-      expose(8080)
+      runRaw(
+        s"wget -q https://download-keycdn.ej-technologies.com/jprofiler/$JProfilerVersion.tar.gz -P /tmp/ && " +
+          s"tar -xzf /tmp/$JProfilerVersion.tar.gz -C /usr/local &&" +
+          s" rm /tmp/$JProfilerVersion.tar.gz"
+      )
+      env(
+        "JPAGENT_PATH",
+        "-agentpath:/usr/local/jprofiler9/bin/linux-x64/libjprofilerti.so=nowait"
```

Comment:
Do we need any new env var to point at the conf file? Or will we need to specify it manually whenever we start up `jprofiler`?

Reply:
we specify it when we run jpenable in offline mode

jpenable -o -c /etc/clio-profile.xml